core/src/main/scala/cats/data/IndexedReaderWriterStateT.scala

```diff
@@ -109,6 +109,13 @@ final class IndexedReaderWriterStateT[F[_], E, L, SA, SB, A](val runF: F[(E, SA)
       }
     }
 
+  def flatTap[SC, B](f: A => IndexedReaderWriterStateT[F, E, L, SB, SC, B])(
```

Comment:
This one as well.

Reply:
This version allows changing the state type. `Monad` instance exists for SA = SB only.